packages/prisma/src/prisma/schema.prisma

```diff
@@ -53,6 +53,8 @@ model User {
   lastLoginAt         DateTime?
   deletionToken       String?
   deletionRequestedAt DateTime?
+  loginToken          String?
```

Comment:
@rschlaefli These changes still require a prisma mutation for the database to be updated correctly. Other than that, this PR is ready to be merged as well.

Reply:
Migration has been applied